### DISCUSSION TITLE
Homematic default Instanz an HomeKit anbinden?

### DISCUSSION THREAD

**Discussion by akula3** [thkl/hap-homematic#338] [Q&A] [answered]
Ich würde gerne die default Instanz direkt ins HomeKit übernehmen, anstatt der einzelnen Raum Instanzen.
Die App kann die default Instanz anhand des QR Codes auch finden, kann aber nicht verbinden.
Geht es überhaupt?

**Comment by jb-home** [comment]
Ja, das geht.
Es hat aber schon einen Sinn und Zweck pro Raum eine eigene Instanz anzulegen.
Inwieweit funktioniert das Verbinden denn nicht?

**Comment by akula3** [comment]
Eine "Raum" Instanz (in diesem Fall "Diele") lässt sich in Home App anbinden und steuern.
Die default Instanz wird per QR Code als Bridge erkannt, der Verbindungsversuch wird gestartet und nach einer Weile es bricht ab mit der Meldung "Gerät wurde nicht gefunden".
Der entsprechende Port ist in der Raspberrymatic freigegeben.
![grafik](https://user-images.githubusercontent.com/58461189/104835996-a1f7fa80-58aa-11eb-8099-e2d17e34c505.png)

**Reply by jb-home** [reply]
Da deine Konfiguration bisher noch sehr übersichtlich ist, würde ich die Vorschlagen den persist Ordner mal zu löschen. Siehe dazu https://github.com/thkl/hap-homematic/issues/253

Vorher noch 'HomeMatic Diele' aus HomeKit löschen. So das du nichts mehr von HAP-Homematic in HomeKit hast!
Ich vermute, dass am Anfang die Verbindung wegen der Firewall nicht funktioniert hatte und das nachträglich öffnen der Firewall die Bridge in einem blöden Zustand hinterlassen hat.

**Reply by akula3** [reply]
Das hat leider nichts gebracht...

**Comment by akula3** [comment, marked as answer]
Hab's rausgefunden!
Die Kopplung **_darf nicht_** per QR Code erfolgen, sondern unbedingt per PinCode.

**Reply by jb-home** [reply]
Das sollte sich @thkl mal anschauen.

Generell sollte es keinen Unterschied machen, ob per Pin Code oder QR Code.
Da war aber mal was mit den Instanznamen und groß und klein Schreibung.
